solr/solr-ref-guide/src/other-parsers.adoc

```diff
@@ -591,36 +591,95 @@ The hash range query parser uses a special cache to improve the speedup of the q
+
+=== Parameters
+
+This query parser takes the following parameters:
+
+`from`::
+The name of a field which contains values to look for in the "to" field.
+Can be single or multi-valued, but must have a field type compatible with the field represented in the "to" field.
+This parameter is required.
+
+`to`::
+The name of a field whose value(s) will be checked against those found in the "from" field.
+Can be single or multi-valued, but must have a field type compatible with the "from" field.
+This parameter is required.
+
+`fromIndex`::
+The name of the index to run the "from" query (`v` parameter) on and where "from" values are gathered.
+Must be located on the same node as the core processing the request.
+This parameter is optional; it defaults to the value of the processing core if not specified.
+See <<Joining Across Collections,Joining Across Collections>> below for more information.
+
+`score`::
+An optional parameter that instructs Solr to return information about the "from" query scores.
+The value of this parameter controls what type of aggregation information is returned.
+Options include `avg` (average), `max` (maximum), `min` (minimum), `total` (total), or `none`.
++
+If `method` is not specified but `score` is, then the `dvWithScore` method is used.
+If `method` is specified and is not `dvWithScore`, then the `score` value is ignored.
+See the `method` parameter documentation below for more details.
 
-=== Join Parser Scoring
 
-You can optionally use the `score` parameter to return scores of the subordinate query. The values to use for this parameter define the type of aggregation, which are `avg` (average), `max` (maximum), `min` (minimum) `total`, or `none`.
+`method`::
+An optional parameter used to determine which of several query implementations should be used by Solr.
+Options are restricted to: `index`, `dvWithScore`, and `topLevelDV`.
+If unspecified the default value is `index`, unless the `score` parameter is present which overrides it to `dvWithScore`.
+Each implementation has its own performance characteristics, and users are encouraged to experiment to determine which implementation is most performant for their use-case.
+Details and performance heuristics are given below.
++
+`index` the default `method` unless the `score` parameter is specified.
+Uses inverted index structures to process the request.
+Performance scales linearly with the number of values matched in the "from" field.
+Consider this method when the "from" query matches few documents, when the "to" side returns a large number of documents, or when sporadic post-commit slowdowns cannot be tolerated (this a disadvantage of other methods that `index` avoids).
 
-.Score parameter and single value numerics
++
+`dvWithScore` returns an optional "score" statistic alongside result documents.
+Uses docValues structures if available, but falls back to uninverted values when necessary.
```

Comment:
"uninverted values" -> "the field cache"
I believe "uninvert" is more of an internal word whereas publicly we say "field cache"?